flink-runtime/src/main/java/org/apache/flink/runtime/io/network/netty/PartitionRequestQueue.java

```diff
@@ -124,7 +124,7 @@ private void enqueueAvailableReader(final NetworkSequenceViewReader reader) thro
 		return availableReaders;
 	}
 
-	public void notifyReaderCreated(final NetworkSequenceViewReader reader) {
+	void notifyReaderCreated(final NetworkSequenceViewReader reader) {
```

Comment:
nit: are the changes in this file part of previous commit or a separate hotfix/refactor/clean up?

Reply:
ok, let's not destroy git history with this one change

Reply:
![Nobody would like this happen](https://media.giphy.com/media/HhTXt43pk1I1W/giphy.gif)